builder/bundles.go

```diff
@@ -212,18 +194,21 @@ func addUpdateBundleSpecialFiles(b *Builder, bundle *bundle) {
 
 func resolveFilesForBundle(bundle *bundle, packagerCmd []string) error {
 	bundle.Files = make(map[string]bool)
-	for p := range bundle.DirectPackages {
-		queryString := merge(packagerCmd, "repoquery", "-l", "--quiet", p)
-		outBuf, err := helpers.RunCommandOutput(queryString[0], queryString[1:]...)
-		if err != nil {
-			return err
-		}
-		for _, f := range strings.Split(outBuf.String(), "\n") {
-			if len(f) > 0 {
-				addFileAndPath(bundle.Files, resolveFileName(f))
-			}
+
+	queryString := merge(packagerCmd, "repoquery", "-l", "--quiet")
+	for p := range bundle.AllPackages {
```

Comment:
Here we are having to convert the map to an array again. We probably do this far too often. Might be worth keeping parallel data structures.